client/my-sites/purchases/navigation.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import SectionNav from 'components/section-nav';
+import NavTabs from 'components/section-nav/tabs';
+import NavItem from 'components/section-nav/item';
+
+export default function PurchasesNavigation( {
+	sectionTitle,
+	siteSlug,
+}: {
```

Comment:
Hmm, didn't know our linting would let this through 🤔

Reply:
@krymson24: could you expound? I'm still pretty new to TypeScript, but this seems to be pretty standard syntax.

Reply:
This just looked weird to me -- but I think it's correct. Carry on!